### PR TITLE
chore(regTests): add basic module logging and print when tests fail

### PR DESCRIPTION
**Pull request by kostasrim**
* add vmodule (dragonfly_connection, accept_server, listener_interface) level 1 logging for df instances
* print generated logs when an instance fails to stop

### CODE REVIEW

File: tests/dragonfly/instance.py [hunk 1]
```diff
@@ -233,6 +236,15 @@ def get_logs_from_psutil(self) -> List[str]:
                 rv.append(file.path)
         return rv
 
+    def print_info_logs_to_debug_log(self):
+        logs = self.log_files
+        for log in logs:
+            if "INFO" in log:
+                with open(log) as file:
+                    logging.debug(f"====== LOG name {log} ======")
```
Comment (romange) [inline tests/dragonfly/instance.py:244]: nice. small suggestion to make it more catchy, and fun: github actions support terminal coloring. why not use term colors to color background/foreground to allow easier finding this by eye? 

also with the header  - to make it more "lively", easier to find - to add emojis lilke 🪵🪵🪵🪵🪵🪵🪵  (tree log) or something else ?

File: tests/dragonfly/instance.py [hunk 2]
```diff
@@ -279,6 +291,8 @@ def create(self, **kwargs) -> DflyInstance:
         args = {**self.args, **kwargs}
         args.setdefault("dbfilename", "")
         args.setdefault("use_zset_tree", None)
+        vmod = "dragonfly_connection=1,accept_server=1,listener_interface=1"
+        args.setdefault("vmodule", vmod)
```
Comment (romange) [inline tests/dragonfly/instance.py:295]: I suggest also adding `main_service=1` and also replication related files: rdb_save=1,replica=1

File: tests/dragonfly/instance.py (hunk 1 above)
Comment (kostasrim) [reply tests/dragonfly/instance.py:244]: > nice. small suggestion to make it more catchy and fun: github actions support terminal coloring. why not use term colors to color background/foreground to allow easier finding this by eye?

+1, we already do that for `stdout` by replacing log level/date with port/arrow (courtesy of :man_cook: Vlad)

File: tests/dragonfly/instance.py [hunk 3]
```diff
@@ -233,6 +236,17 @@ def get_logs_from_psutil(self) -> List[str]:
...
 
+    def print_info_logs_to_debug_log(self):
+        logs = self.log_files
+        sed_format = f"s/[^ ]*/{self.port}{Colors.next()}➜{Colors.CLEAR}/"
+        sed_cmd = ["sed", "-e", sed_format]
+        print("kostas")
+        for log in logs:
+            if "INFO" in log:
+                with open(log) as file:
+                    print(f"🪵🪵🪵🪵🪵🪵 LOG name {log} 🪵🪵🪵🪵🪵🪵")
```
Comment (kostasrim) [inline tests/dragonfly/instance.py:247]: @romange I love this :star:

File: tests/dragonfly/instance.py (hunk 1 above)
Comment (kostasrim) [reply tests/dragonfly/instance.py]: falls back to the same coloring vlad introduced. I also replaced the <level/data> with <port/arrow> to be consistent

**Review by dranikpg — APPROVED** [review]

**Pull request merged**